.github/CODEOWNERS

```
@@ -0,0 +1,3 @@
+# Use for automatic reviewer tagging. (anyone with merge rights is encouraged
+# to review and approve PRs they understand)
+*   @NullHypothesis @ypatia @shaunrd0 @snagles
```

I would add @thetorpedodog to this list.
```suggestion
*   @NullHypothesis @ypatia @shaunrd0 @snagles @thetorpedodog
```